packages/rum-core/src/performance-monitoring/capture-navigation.js

```diff
@@ -254,24 +254,21 @@ function getNavigationTimingMarks() {
  * SPEC - https://www.w3.org/TR/paint-timing/
  */
 function getFirstContentfulPaint() {
-  let fcp
   if (isPerfTimelineSupported()) {
-    const entries = PERF.getEntriesByType(PAINT)
-    if (entries.length > 0) {
-      const timing = PERF.timing
-      /**
-       * To avoid capturing the unload event handler effect
-       * as part of the transaction duration
-       */
-      const unloadDiff = timing.fetchStart - timing.navigationStart
-      const fcpEntry = entries.filter(
-        entry => entry.name === FIRST_CONTENTFUL_PAINT
-      )
-      const { startTime } = fcpEntry[0]
-      fcp = unloadDiff >= 0 ? startTime - unloadDiff : startTime
-    }
+    const timing = PERF.timing
+    /**
+     * To avoid capturing the unload event handler effect
+     * as part of the transaction duration
+     */
+    const unloadDiff = timing.fetchStart - timing.navigationStart
+    const fcp = PERF.getEntriesByType(PAINT).reduce((_fcp, entry) => {
```

Comment:
can you add a initial value of `null` to the reduce function as entries can be empty array `[]` and we need an initial value for reduce to work. 

```js
reduce(fcp,entry=> {}, null)
```

Reply:
CLA Signed and initialValue added.